### DISCUSSION TITLE
Bento Draggable Layout

### DISCUSSION THREAD

**Discussion by nomercy360** [clauderic/dnd-kit#1216] [Q&A]
Hi, what is the best way to implement a bento.me like behaviour? which package utilities can be used, which example will more close?
Kind of Draggable Dashboard layout where i can have items with different width and height and they will fit inside some area

[63ea5abefbeb7e96b8d77a3f_desktop video small-transcode.webm](https://github.com/clauderic/dnd-kit/assets/50191647/eb8e2ff3-d189-412c-8536-3fb3e9203685)

**Comment by taziksh** [comment]
Cool site! Drop the link

**Reply by md-moijul** [reply]
f

**Comment by purple-force** [comment]
react-grid-layout
